libkmod/libkmod.c

```diff
@@ -209,7 +209,7 @@ static char *get_kernel_release(const char *dirname)
 	if (uname(&u) < 0)
 		return NULL;
 
-	if (asprintf(&p, "%s/%s", dirname_default_prefix, u.release) < 0)
```

Comment:
when kmod_new() is called with dirname != NULL, it will just return in the check above, so I don't think we need to change this.